RULES.md

```diff
@@ -1,3004 +1 @@
-
-### runtime_exception_in_loader_error
-
-A [RuntimeException](https://docs.oracle.com/javase/8/docs/api/java/lang/RuntimeException.html) occurred while loading a table. This normally indicates a bug in validator.
-<details>
-
-
-#### Notice fields description
-| Field name 	  | Description                                                   	| Type    |
-|-------------	|---------------------------------------------------------------	|---------|
-| `filename`  	| The name of the file that caused the exception.            	    | String 	|
-| `exception`  	| The name of the exception.                                    	| String 	|
-| `message`    	| The error message that explains the reason for the exception. 	| String  |
-
-#### Affected files
-[All GTFS files supported by the specification.](http://gtfs.org/reference/static#dataset-files)
-</details>
-
-<a name="RuntimeExceptionInValidatorError"/>
-
-### runtime_exception_in_validator_error
-
-A [RuntimeException](https://docs.oracle.com/javase/8/docs/api/java/lang/RuntimeException.html) occurred during validation. This normally indicates a bug in validator code, e.g., in a custom validator class.
-<details>
-
-#### Notice fields description
-| Field name 	| Description                                                   	| Type    |
-|------------	|---------------------------------------------------------------	|---------|
-| `validator` | The name of the validator that caused the exception.            | String 	|
-| `exception` | The name of the exception.                                    	| String 	|
-| `message`   | The error message that explains the reason for the exception. 	| String  |
-</details>
-
-<a name="ThreadExecutionError"/>
-
-### thread_execution_error
-
-An [ExecutionException](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutionException.html) occurred during multithreaded validation.
-<details>
-
-
-#### Notice fields description
-| Field name 	  | Description                                                   	| Type    |
-|-------------	|---------------------------------------------------------------	|---------|
-| `exception`  	| The name of the exception.                                    	| String 	|
-| `message`    	| The error message that explains the reason for the exception. 	| String  |
-</details>
-
-<a name="URISyntaxError"/>
-
-### u_r_i_syntax_error
-
-A string could not be parsed as a URI reference.
-<details>
-
-
-#### Notice fields description
-| Field name 	  | Description                                                   	| Type    |
-|-------------	|---------------------------------------------------------------	|---------|
-| `exception`  	| The name of the exception.                                    	| String 	|
-| `message`    	| The error message that explains the reason for the exception. 	| String  |
-</details>
+The documented list of all notices is now available at https://gtfs-validator.mobilitydata.org/rules.html
```

Comment:
I don't know if it has an incidence, but Brian's change kept the # Implemented notices line at the beginning

Reply:
I put it back.